### PR TITLE
[FLINK-16098] [chinese-translation, Documentation] Translate "Overview" page of "Hive Integration" into Chinese

### PR DESCRIPTION
**Pull request by liuzhixing1006**
Translate "Overview" page of "Hive Integration" into Chinese
jira：https://issues.apache.org/jira/projects/FLINK/issues/FLINK-16098?filter=myopenissues

### CODE REVIEW

**Comment by wuchong** [comment]
Could you help to review this? @lirui-apache

**Comment by liuzhixing1006** [comment]
Is there a problem with this commit? 
Please suggest that I revise it in time. @wuchong

**Comment by JingsongLi** [comment]
Thanks @liuzhixing1006 for contribution, review is on the way.

**Comment by lirui-apache** [comment]
Sorry I overlooked this one. Will take a look.

File: docs/dev/table/hive/index.zh.md [hunk 1]
```diff
@@ -77,29 +77,29 @@ Flink supports the following Hive versions.
...
-- Hive built-in functions are supported in 1.2.0 and later.
-- Column constraints, i.e. PRIMARY KEY and NOT NULL, are supported in 3.1.0 and later.
-- Altering table statistics is supported in 1.2.0 and later.
-- `DATE` column statistics are supported in 1.2.0 and later.
-- Writing to ORC tables is not supported in 2.0.x.
+- Hive内置函数，已在1.2.0及更高版本予以支持。
+- 列约束，也就是PRIMARY KEY 和 NOT NULL，已在3.1.0及更高版本予以支持。
+- 更改表的统计信息，已在1.2.0及更高版本予以支持。
+- `DATE`列统计，已在1.2.0及更高版本予以支持。
+- 2.0.x版本不支持写入ORC表。
```
Comment (lirui-apache) [inline docs/dev/table/hive/index.zh.md]:
```suggestion
- 使用Hive-2.0.x版本时不支持写入ORC表。
```

File: docs/dev/table/hive/index.zh.md [hunk 2]
```diff
@@ -77,29 +77,29 @@ Flink supports the following Hive versions.
...
 
-### Dependencies
+### 依赖项
 
-To integrate with Hive, you need to add some extra dependencies to the `/lib/` directory in Flink distribution
-to make the integration work in Table API program or SQL in SQL Client.
-Alternatively, you can put these dependencies in a dedicated folder, and add them to classpath with the `-C`
-or `-l` option for Table API program or SQL Client respectively.
+要与Hive集成，您需要在Flink下的`/lib/`目录中添加一些额外的依赖关系，
```
Comment (lirui-apache) [inline docs/dev/table/hive/index.zh.md]:
```suggestion
要与Hive集成，您需要在Flink下的`/lib/`目录中添加一些额外的依赖包，
```

File: docs/dev/table/hive/index.zh.md [hunk 3]
```diff
@@ -77,29 +77,29 @@ Flink supports the following Hive versions.
...
+使得集成在Table API的程序或SQL Client中的SQL能够起到作用。
```
Comment (lirui-apache) [inline docs/dev/table/hive/index.zh.md]:
```suggestion
以便通过Table API或SQL Client与Hive进行交互。
```

File: docs/dev/table/hive/index.zh.md [hunk 4]
```diff
@@ -77,29 +77,29 @@ Flink supports the following Hive versions.
...
+或者，您可以将这些依赖项放在专用文件夹中，并分别使用Table API程序或SQL Client的`-C`或`-l`选项将它们添加到classpath中。
 
-Apache Hive is built on Hadoop, so you need Hadoop dependency first, please refer to
+Apache Hive 是基于Hadoop之上构建的, 首先您需要hadoop的依赖，请参考
 [Providing Hadoop classes]({{ site.baseurl }}/ops/deployment/hadoop.html#providing-hadoop-classes).
 
-There are two ways to add Hive dependencies. First is to use Flink's bundled Hive jars. You can choose a bundled Hive jar according to the version of the metastore you use. Second is to add each of the required jars separately. The second way can be useful if the Hive version you're using is not listed here.
+有两种添加Hive依赖项的方法。第一种是使用Flink捆绑Hive的Jar包。您可以根据使用的metastore的版本来选择捆绑的Hive jar。第二个方式是分别添加每个所需的jar包。如果您使用的Hive版本尚未在此处列出，则第二种方法会更适合。
```
Comment (lirui-apache) [inline docs/dev/table/hive/index.zh.md]:
```suggestion
有两种添加Hive依赖项的方法。第一种是使用Flink提供的Hive Jar包。您可以根据使用的metastore的版本来选择对应的Hive jar。第二个方式是分别添加每个所需的jar包。如果您使用的Hive版本尚未在此处列出，则第二种方法会更适合。
```

File: docs/dev/table/hive/index.zh.md [hunk 5]
```diff
@@ -77,29 +77,29 @@ Flink supports the following Hive versions.
...
 
-#### Using bundled hive jar
+#### 使用捆绑的Hive jar
```
Comment (lirui-apache) [inline docs/dev/table/hive/index.zh.md]:
```suggestion
#### 使用Flink提供的Hive jar
```

File: docs/dev/table/hive/index.zh.md [hunk 6]
```diff
@@ -77,29 +77,29 @@ Flink supports the following Hive versions.
...
+
+下表列出了所有可捆绑的hive jars。您可以在Flink发行版的`/lib/` 目录中去选择一个。
```
Comment (lirui-apache) [inline docs/dev/table/hive/index.zh.md]:
```suggestion
下表列出了所有可用的Hive jar。您可以选择一个并放在Flink发行版的`/lib/` 目录中。
```

File: docs/dev/table/hive/index.zh.md [hunk 7]
```diff
@@ -259,15 +259,14 @@ Please find the required dependencies for different Hive major versions below.
 </div>
 </div>
 
-If you use the hive version of HDP or CDH, you need to refer to the dependency in the previous section and select a similar version.
+如果使用hive的HDP或CDH版本，则需要参考上一节中的依赖项并选择一个类似的版本。
```
Comment (lirui-apache) [inline docs/dev/table/hive/index.zh.md]:
```suggestion
如果使用Hive的HDP或CDH版本，则需要参考上一节中的依赖项并选择一个类似的版本。
```

File: docs/dev/table/hive/index.zh.md [hunk 8]
```diff
@@ -259,15 +259,14 @@ Please find the required dependencies for different Hive major versions below.
...
 
-And you need to specify selected and supported "hive-version" in yaml, HiveCatalog and HiveModule.
+并且您需要在yaml，HiveCatalog和HiveModule中指定选择的和受支持的“ hive-version”。
```
Comment (lirui-apache) [inline docs/dev/table/hive/index.zh.md]:
```suggestion
并且您需要在定义yaml文件，或者创建HiveCatalog和HiveModule时，指定一个支持的“ hive-version”。
```

File: docs/dev/table/hive/index.zh.md [hunk 9]
```diff
@@ -259,15 +259,14 @@ Please find the required dependencies for different Hive major versions below.
...
 
-#### Program maven
+### 程序maven依赖
```
Comment (lirui-apache) [inline docs/dev/table/hive/index.zh.md]:
```suggestion
### Maven依赖
```

File: docs/dev/table/hive/index.zh.md [hunk 10]
```diff
@@ -294,18 +293,15 @@ You're supposed to add dependencies as stated above at runtime.
 </dependency>
 {% endhighlight %}
 
-## Connecting To Hive
+## 连接到Hive
 
-Connect to an existing Hive installation using the [catalog interface]({{ site.baseurl }}/dev/table/catalogs.html) 
-and [HiveCatalog]({{ site.baseurl }}/dev/table/hive/hive_catalog.html) through the table environment or YAML configuration.
+通过表环境或者YAML配置，使用 [catalog interface]({{ site.baseurl }}/dev/table/catalogs.html) 和[HiveCatalog]({{ site.baseurl }}/dev/table/hive/hive_catalog.html)连接到现有的Hive集群。
```
Comment (lirui-apache) [inline docs/dev/table/hive/index.zh.md]:
```suggestion
通过TableEnvironment或者YAML配置，使用 [catalog interface]({{ site.baseurl }}/dev/table/catalogs.html) 和[HiveCatalog]({{ site.baseurl }}/dev/table/hive/hive_catalog.html)连接到现有的Hive集群。
```

**Comment by lirui-apache** [comment]
@liuzhixing1006 Thanks for the PR! I left some comments.

**Comment by liuzhixing1006** [comment]
@JingsongLi Thanks for your advice, I have improved the problem.
But I had a problem updating the branch,I use the following in order:

1.git add -A
2.git commit -am [FLINK-16098] [chinese-translation, Documentation] Translate "Overview" page of "Hive Integration" into Chinese
3.git fetch upstream master
4.git merge upstream/master
5.git rebase -i master
5.git push origin docFix

Then in the pull request you find that there are a lot of unrelated commits。
Is there something wrong with me？
Do I need to close the current pullrequest and create a new one？
Thanks~

**Comment by JingsongLi** [comment]
Hi @liuzhixing1006 , you can not use merge. You should use rebase only.

**Comment by liuzhixing1006** [comment]
Thank you @JingsongLi  for bringing this problem to my attention, There's something wrong with this branch, I created a new pr [https://github.com/apache/flink/pull/11455](New PR).
I'm sorry to have caused so much trouble.

**Pull request closed without merging**